### ISSUE TITLE
CMS Team Sprint 94 Plan (9.27.2023 - 10.10.2023)

### ISSUE THREAD

**Issue by BerniXiongA6** [department-of-veterans-affairs/va.gov-cms#15323]
## Sprint 94 Goals & Priorities
_At the end of this sprint, what do we want to have accomplished/delivered?_
- [x] Completed next research efforts on enhancing the KB self-service experience: #15249 
- [x] Completed presentation on Drupal 10 updates for VAMC editors office hours 10/11: #15420
- [x] Identified opportunities for Datadog monitoring expansion: #15406

**Notes:** 
<details><summary>**Sprint 94 Prioritization Notes**</summary>

- Capacity: All Points CMS Team Can Take On in S94 = 52.5
> All Eng = 35.5
> All UX = 12.6
> All Support = 2.6
> a11y = 1.8

</details>

### Top Priority
The following list in this Top Priority section are the issues to work on first. These are ordered on the top of the To Do (Current Sprint) pipeline on the board:

#### D10 - Runbook Development (Q4 Goal) 
Engineering
- [x] (5) #15237

#### D10 - Change Management Efforts (VAMC Editors) - Due 10/6 for meeting on 10/11 (Q4 Goal) 
a11y & Training
- [x] (5) #15420 (Sprint Goal)
- [x] (1.5) #15154 (Laura's KB article already in progress on rich text editor to be reviewed by Training and Product)

#### Datadog Monitoring Expansion (Q4 Goal)
- [x] (3) #15406 (Sprint Goal)
- [ ] (5) #15307 (if didn't get to it, can we carry over to S95?)
- [x] (5) #13984 *blocking # 13985
- [x] (3) #13985 *start after # 13984

#### Alt Text / Client Side Validation Implementation (Q4 Goal)
- none for S94
-
#### PACT ACT Coverage
Engineering, Product, Support
- [x] (2) #15462

#### Defects, Hurdles
- [x] (3) #15447

#### Keep the lights on (KTLO)
Drupal
- [ ] (1) #15439 (looks like we didn't get any in Sprint 94 - BX 10/10)

DevOps
- [x] (3) #15426
- [ ] (8) carry-over from S93 #14350 (probably need to carry over again - BX 10/10)

### Medium Priority
The following list in this Medium Priority section are the issues to work on after the above have been completed. These are also ordered to follow the top priority issues in the To Do (Current Sprint) pipeline on the board:

#### Knowledge-base Enhancements #14069 
UX Research
- [x] (5) #15249 (Sprint Goal)

#### CMS Governance by Discipline/Function & CMS Collab Cycle 
Drupal
- [x] (3) #15308 

UX + Product
- [x] #15438

#### Improvements to content release system #14016
Drupal
- [ ] (8) #6434 - Nate to work with other DevOps on this (if unable to get to in S94, carry over to Sp5?)

UX Design
- [x] (3) #12359 

#### Help clients, stakeholders hit their objectives.
Product / Delivery
- [ ] (1) #15257 (will carryover and finish at the beginning of S95 - BX 10/10)
- [ ] (1) #15328 (will carryover and finish at the beginning of S95 - BX 10/10)

#### Pulled in Post-Sprint-Planning
DevOps
- [x] (2) #15383
- [x] (3) #15220
- [x] (5) #15302
- [x] (3) #15406
- [x] (3) #15614
- [x] (3) #14924
- [x] (3) #15482
- [x] (1) #15362

Drupal
- [x] #14293
- [x] #15112

### Not Included in Sprint 94 Commitments (Pull from this list ONLY if Top and Medium Priority issues are completed)
The following list in this Ready for Sprint section are the issues that can be worked on after the Top and Medium Priority issues have been completed. These are ordered in the Ready for Sprint pipeline on the board:

#### Open DevSecOps Tickets (carry over to S95 low priority candidates)
- [ ] (5) #14492
- [ ] (8) #13834
- [ ] (8) #14971
- [ ] (5) #15301
- [ ] (5) #15307
- [ ] (5) #13975
- [ ] (3) #11632
- [ ] (8) #7674
- [ ] (5) #7673
- [ ] (5) #11629

#### Open Drupal Engineering Tickets  (carry over to S95 low priority candidates)
- [ ] (2) #14765 
- [ ] (8) #13639
- [ ] (3) #12674

#### Drupal Engineering Tickets (Need Refining)  (carry over to S95 for Backlog Refinement candidates)
- [ ] #12522 (needs refining)
- [ ] #10119 (needs refining)
- [ ] #13982 (needs refining)
- [ ] #13697 (needs refining)
- [ ] #10356 (needs refining)
- [ ] #15419 (needs refining)
- [ ] #15410 (needs refining)

**Comment by EWashb** [comment]
@BerniXiongA6 Do you have a ticket ready for the prep for VHA DM Office Hours for D10 updates on 10/11/23? If not, this is a friendly reminder to account for that in this sprint :)

**Comment by BerniXiongA6** [comment]
Closed out Sprint 94 at 134% of sprint capacity plan (70.5 pts).
